requirements/base.txt

```diff
@@ -55,13 +55,13 @@ pyyaml==6.0
     # via code-annotations
 six==1.16.0
     # via -r requirements/base.in
-stevedore==3.5.0
+stevedore==4.0.0
```

Comment:
Not any major incompatible change for now in the version [7.0.0](https://github.com/simplistix/testfixtures/blob/master/CHANGELOG.rst#700-6-jul-2022)